collection-scripts/gather_network_logs

```diff
@@ -26,6 +26,59 @@ if [ "$NETWORK_TYPE" == "openshiftsdn" ]; then
         oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
         PIDS+=($!)
     done
+elif [ "$NETWORK_TYPE" == "kuryr" ]; then
+      CONTROLLER_POD=$(oc -n openshift-kuryr get pods --no-headers -o custom-columns=':metadata.name' -l app=kuryr-controller | tail -1)
+      oc -n openshift-kuryr logs $CONTROLLER_POD > \
+      ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_log &
+      PIDS+=($!)
+
+      CONTROLLER_RESTART_COUNT="${@:-$(oc get po $CONTROLLER_POD -n openshift-kuryr -o jsonpath={.status.containerStatuses[*].restartCount})}"
+      if [ "$CONTROLLER_RESTART_COUNT" -ne "0" ]; then
+          oc -n openshift-kuryr logs $CONTROLLER_POD --previous > ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_previous_log &
+          PIDS+=($!)
+      fi
+
+      NODES="${@:-$(oc get nodes --no-headers -o custom-columns=':metadata.name')}"
+      for NODE in $NODES; do
+          CNI_POD=$(oc -n openshift-kuryr get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$NODE -l app=kuryr-cni)
+          oc -n openshift-kuryr logs $CNI_POD > ${NETWORK_LOG_PATH}/${CNI_POD}_log &
+          PIDS+=($!)
+          RESTART_COUNT="${@:-$(oc get po $CNI_POD -n openshift-kuryr -o jsonpath={.status.containerStatuses[*].restartCount})}"
+          if [ "$RESTART_COUNT" -ne "0" ]; then
+              oc -n openshift-kuryr logs $CNI_POD --previous > ${NETWORK_LOG_PATH}/${CNI_POD}_previous_log &
+              PIDS+=($!)
+          fi
+          oc exec -n openshift-kuryr $CNI_POD -- bash -c 'for pid in $(find /host_proc/[1-9]*/ns/net | cut -d/ -f3); do\
+              nsenter --net=/host_proc/$pid/ns/net ip -d addr;\
```

Comment:
Ah, it's worth to print the namespace name so we can find the corresponding request in kuryr-daemon logs.